app.rb

```diff
@@ -1,8 +1,56 @@
 require 'sinatra/base'
+require_relative 'lib/player'
+require_relative 'lib/game'
+
 class RockPaperScissors < Sinatra::Base
+  enable :sessions
+
   get '/test' do
     'test page'
   end
 
+  get '/' do
+    erb :index
+  end
+
+  get '/names' do
+    $player_one = Player.new(params[:player_one])
+    $player_two = Player.new(params[:player_two])
+    redirect to('/play')
+  end
+  
+  get '/pvp-names' do
+    erb :pvp_names
+  end
+
+  get '/vs-ai-name' do
+    erb :vs_ai_name
+  end
+
+  get '/play' do
+    @player_one = $player_one
+    @player_two = $player_two
+    if @player_two.name == 'computer'
+      erb :play
+    else
+      erb :pvp_play
+    end
+  end
```

Comment:
This logic would be better in the model.

Reply:
Thanks for the feedback, really useful :)